.github/workflows/issue-add-to-parent-project.yml

```diff
@@ -0,0 +1,134 @@
+name: When epic issues changed, check if part of specified child projects and update on specified parent project
```

Comment:
If this is not a generic rule and is specific to Platform UX and the squads that make up Platform UX, it might be helpful to have that in the name here and in the yml file name. 

If another team with the same parent/child hierarchy in projects wants this action, would they have to copy this file and add their own parent/child projectIDs?

Reply:
Platform UX specific now as an experiment but the plan would be to expand it if it works.
Yep they would have to copy and replace some values so maybe I should rename the doc to make it platform ux specific then for now.

Reply:
Renamed and updated the description as well. Thanks!